tests/jballerina-unit-test/src/test/resources/test-src/types/xml/xml_text_to_string_conversion.bal

```diff
@@ -81,9 +70,22 @@ function getXMLText() returns 'xml:Text {
     return xml `my world`;
 }
 
+function testXMLWithAngleBrackets() {
```

Comment:
Sorry, missed this earlier, but is this test file and the class the correct place to add this test? This is to test xml:Text to string conversion, right?

Reply:
Agree, will move to langlib tests